meshtastic/interfaces/ble/lifecycle_receive_runtime.py

```
@@ -0,0 +1,206 @@
+        """Validate start preconditions and create a staged receive thread."""
+        iface = self._iface
+        with iface._state_lock:
+            if iface._closed or not iface._want_receive:
+                logger.debug(
+                    "Skipping receive thread start (%s): interface is closing/stopped.",
+                    name,
+                )
+                return None, None
+            existing = iface._receiveThread
+            existing_start_pending = bool(getattr(iface, "_receive_start_pending", False))
+            existing_ident, existing_is_alive = (
```

Ruff (0.15.6)

[warning] Unpacked variable `existing_ident` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)